OxfordBodleian/BodNewMSS/BDLaethf35.xml

```diff
@@ -146,14 +146,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <p>A long tear has been repaired by sewing on <locus target="#ii"/>.</p>
                                 </support>
                                 <extent>
-                                    <measure unit="leaf" quantity="114">iv+110</measure>
-                                    <!--format?-->
-                                    <measure unit="leaf" type="blank">9</measure>
-                                    <!--pages-->
-                                    <locus from="i" to="iv"/>
-                                    <locus target="#80"/>
+                                    <measure unit="leaf" quantity="114">114</measure>
+                                    <measure unit="quire" quantity="12">12</measure>
+                                    <measure unit="page" type="blank">17</measure>
+                                    <locus from="ir" to="ivv"/>
```

Comment:
I do agree with Jonas, it might be difficult to understand, but if it is according to guidelines is ok.

Reply:
Your system is much better! So far, it has not been used. I would be open to updating all descriptions (in a separate PR) accordingly, but this would need everyone's agreement. What do you think?

Reply:
or using capital letters for Roman numbers like "Ir" or "IVv"?

Reply:
From a certain poit of view, Eliana's solution seems to me preferable, because the dot between the leaf number and the recto/verso does not appear in the Arabic number (we do not have fol. 2.ra), nor it is part of the original Roman numbering system. I would therefore have a slight preference for indicating Roman numerals in capital letters (IIv, IVv), unless there are other cons that escape me at the moment.
On the other way, the distinction between, e.g.,  fol. _i_ verso and _iv_ is much clearer with Jonas' systema (i.v against iv)

Reply:
I was thinking that we use capital Roman numerals for quires… but that shouldn't really matter here, I suppose. Maybe there could be a risk of confusion between "1v" and "Iv", depending on the font?